src/Services/Air/templates/AIR_AVAILABILTIY_REQUEST.handlebars.js

```diff
@@ -11,9 +11,7 @@ module.exports = `
                 <com:OverridePCC ProviderCode="{{provider}}" PseudoCityCode="{{emulatePcc}}"/>
             {{/if}}
             {{#nextResultReference}}
-                <com:NextResultReference ProviderCode="{{provider}}" >
-                    {{nextResultReference}}
-                </com:NextResultReference>
+                <com:NextResultReference ProviderCode="{{../provider}}">{{{.}}}</com:NextResultReference>
```

Comment:
It's not self-explanatory what the result of `{{{.}}}` will be, could you elaborate on that, please?

Reply:
skip html escaping, as `nextResultReference` is a base64 string which might contain '=' symbol